src/main/webapp/app/iris/settings/iris-settings-update/iris-common-sub-settings-update/iris-common-sub-settings-update.component.ts

```
@@ -64,6 +83,23 @@
         }
     }
 
+    loadCategories() {
+        if (this.settingsType === this.COURSE) {
+            this.courseManagementService.findAllCategoriesOfCourse(this.courseId!).subscribe({
+                next: (response: HttpResponse<string[]>) => {
+                    this.categories = this.exerciseService
+                        .convertExerciseCategoriesAsStringFromServer(response.body!)
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)